python/cudf/cudf/core/indexing.py

```diff
@@ -113,7 +116,19 @@ class _SeriesLocIndexer(object):
     def __init__(self, sr):
         self._sr = sr
 
-    def __getitem__(self, arg):
+    def __getitem__(self, arg: Any) -> Union[ScalarLike, DataFrameOrSeries]:
+        if isinstance(arg, (cudf.MultiIndex, pd.MultiIndex)):
+            if isinstance(arg, pd.MultiIndex):
+                arg = cudf.from_pandas(arg)
+        elif isinstance(self._sr.index, cudf.MultiIndex):
+            result = self._sr.index._get_row_major(self._sr, arg)
+            if (
+                isinstance(arg, tuple)
+                and len(arg) == self._sr._index.nlevels
+                and not any((isinstance(x, slice) for x in arg))
```

Comment:
What is `arg` in this case? Could it be a GPU backed object?

Reply:
A tuple (see 2 lines above) for indexing into the levels of a MultiIndex. Something like `s.loc[('a', 'b', slice(None), 'd')]`